### ISSUE TITLE
Cookie consent mechanism

### ISSUE THREAD

**Issue by epennifold** [exeGesIS-SDM/ELM#33]
As SHINE/NRHE.

**Comment by CrispinF** [comment]
@sarahpoppy would it be a 100% requirement to use the same cookie control mechanism as CS and NRHE, or can the existing simpler mechanism suffice for now? 
NB this item is only "extra small" if it can. A lot more if it cannot.

**Comment by sarahpoppy** [comment]
What is the existing "simpler mechanism"? @CrispinF

**Comment by CrispinF** [comment]
@sarahpoppy if you visit https://elm-hefer.esdm.co.uk/ you will see the default cookie warning. If you've already been there and accepted it, tri it in another browser, or cler cookies and try again.
Obviously the "Learn More" page is to be completed with specifics, and the precise text of the banner can also be easily changed. What this does not provide is the ability to set different permissions for different categories of cookies, but in practice this serves to allow or disallow _non-essential_ cookies, so it amounts to the same thing. That is, this mechanism will block or allow analytics cookies, while also leading users to information about all cookies and privacy policy.

**Comment by CrispinF** [comment]
I'm not 100% certain about this, as it's open to some interpretation, but I believe that if we only use "essential" cookies on this site, there should be no cookie consent mechanism required. (Noting that these mechanisms never allow you to switch off essential cookies other, than by not using the site). 
The only cookies we have by default are essential, and this would only change if we add things like Google Analytics. So if we do not add Analytics, are you happy @sarahpoppy that we switch off the cookie consent mechanism? This helps remove friction for users.
(There would still be a need to include information on privacy policy and cookies, on the /privacy page.)

**Comment by sarahpoppy** [comment]
Yes happy to agree that approach, that we switch off the cookie control mechanism, as long as you can provide info on the essential cookies on the privacy page.  We can always implement later if required.

**Comment by CrispinF** [comment]
Switched off on production site. 
Privacy page still to be written.
